services/common/src/redux/utils/helpers.ts

```diff
@@ -109,16 +109,15 @@ export const formatDateTimeUserTz = (dateTime: Date | string): string => {
 
 export const timeAgo = (dateTime, unit = "day") => {
   const startDate = dateTime;
-  const endDate = new Date().toUTCString();
```

Comment:
Was the issue with the test spec going between 9 to 10 because endDate was created as utc string?

Reply:
Exactly! End date being converted to UTC, whereas startDate was not